<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>581</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="todoListPage">
     <widget class="QTableView" name="ToDoList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>550</width>
        <height>530</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mapPage">
     <widget class="QLabel" name="Map">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>550</width>
        <height>530</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(130, 156, 208);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="imageDownloadPage">
     <widget class="QLabel" name="imageDownload">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>550</width>
        <height>530</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(130, 156, 208);
border: 5px;
border-color: rgb(71, 133, 89);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="localHourLCD">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(235, 235, 235)</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="localMinuteLCD">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(235, 235, 235)</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="localSecondLCD">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(235, 235, 235)</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="Weather">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 156, 208);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="colonOne">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>20</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="colonTwo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>20</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="colonOne_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>20</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="colonTwo_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>20</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="worldHourLCD">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(235, 235, 235);</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="worldMinuteLCD">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(235, 235, 235);</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="worldSecondLCD">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(235, 235, 235);</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QRadioButton" name="page1">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>540</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="page2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>540</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="page3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>540</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
